### DISCUSSION TITLE
Does the Istiod use envoy or essentially be envory proxy?

### DISCUSSION THREAD

**Discussion by pyhackr-io** [istio/istio#54070] [Q&A] [answered]
Hi Community, 

Based on material on Istio accessible on the Internet from various organisation who have used the the Istio API to enhance or provide additional functionality to the "Standard OSS" Istio saying that istiod is essentially an envoy proxy. 

My question :  Is this the case  or is it not that simple? 

 p.s. Improving my knowledge on Istio any material/docs to help would be greatly appreciated. 
 
 thank you in advance
 Pankaj

**Comment by howardjohn** [comment, marked as answer]
Istio is not _an_ Envoy proxy. Rather, it is a combination of deployments of Envoy proxies + a control plane, Istiod, which configures them. Diagram in https://istio.io/latest/docs/ops/deployment/architecture/ may help

**Reply by pyhackr-io** [reply]
Thank you. Follow up question. Istiod = pilot+ citadel+galley. I see in the readme.md the links to pilot + citadel(security) directories.  Would my assumption be correct that galley is part of pilot based on the description ? 
Thank you
